packages/react-vapor/src/components/drop/DropPod.tsx

```diff
@@ -83,8 +83,8 @@ class RDropPod extends React.PureComponent<IRDropPodProps, IDropPodState> {
             };
         } else {
             this.lastPosition = {
-                position: DropPodPosition.bottom,
-                orientation: DropPodPosition.left,
+                position: DropPodPosition?.bottom,
+                orientation: DropPodPosition?.left,

```

Comment:
You can remove the `?` here, `DropPodPosition` is a `const` with the following keys (which will always be defined).
```ts
export const DropPodPosition = {
    bottom: 'bottom',
    top: 'top',
    left: 'left',
    right: 'right',
};
```

Reply:
quite true